test/integration_test/storkctl_migrationschedule_test.go

```diff
@@ -51,25 +52,26 @@ func createDefaultAsyncMigrationScheduleTest(t *testing.T) {
 		"cluster-pair": asyncDrClusterPair,
 		"namespaces":   defaultNs,
 	}
-	createMigrationScheduleTest(t, testrailID, cmdArgs, "default-async-migration-schedule.yaml", defaultNs)
+	createMigrationScheduleTest(t, testrailID, cmdArgs, "default-async-migration-schedule.yaml", defaultNs, "")
 }
 
 func createCustomAsyncMigrationScheduleTest(t *testing.T) {
 	testrailID := 93399
+	intervalMinutes := "25"
```

Comment:
nit..no need to create a variable here. We can treat it like other variables in this same call.

Reply:
I just created the variable because it is being passed in 2 places.
While creating the migrationschedule and to the validate function